css/css-flexbox/flexbox-align-self-baseline-compatability.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE>
```

Comment:
@mrobinson @Loirooriol  quirky doc-type causing failures in other engines.

Reply:
Thanks for the heads up. This should be fixed now.